src/main/java/de/tum/in/www1/artemis/web/rest/dto/standardizedCompetency/StandardizedCompetencyRequestDTO.java

```diff
@@ -12,6 +13,6 @@
  * DTO used to send requests regarding {@link StandardizedCompetency} objects.
  */
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record StandardizedCompetencyRequestDTO(@NotNull @Size(min = 1, max = StandardizedCompetency.MAX_TITLE_LENGTH) String title,
-        @Size(max = StandardizedCompetency.MAX_DESCRIPTION_LENGTH) String description, CompetencyTaxonomy taxonomy, @NotNull Long knowledgeAreaId, Long sourceId) {
+public record StandardizedCompetencyRequestDTO(@NonNull @Size(min = 1, max = StandardizedCompetency.MAX_TITLE_LENGTH) String title,
+        @Size(max = StandardizedCompetency.MAX_DESCRIPTION_LENGTH) String description, CompetencyTaxonomy taxonomy, @NonNull Long knowledgeAreaId, Long sourceId) {
```

Comment:
Same here, `title` and `knowledgeAreaId` being annotated with `NonNull` seems to cause problems in the tests (the corresponding tests pass once I revert the annotations here and in KnowledgeAreaRequestDTO)